.github/workflows/build_and_test.yml

```diff
@@ -89,6 +89,9 @@ jobs:
       # In order to fetch changed files
       with:
         fetch-depth: 0
+    - uses: n1hility/cancel-previous-runs@v2
```

Comment:
This one seems to be not allowed as of now.
![Screen Shot 2021-01-08 at 2 57 07 PM](https://user-images.githubusercontent.com/9700541/104073315-36cf6980-51c2-11eb-9070-9e811c5720fa.png)